packages/bpk-component-image/src/BpkImage.js

```diff
@@ -56,22 +75,23 @@ Image.defaultProps = {
   hidden: false,
 };
 
-class BpkImage extends React.Component {
-  constructor(props) {
-    super(props);
+class BpkImage extends React.Component<BpkImageProps> {
+  onImageLoad: Function;
+  placeholder: any;
+  static defaultProps: {};
 
+  constructor(props: BpkImageProps) {
+    super(props);
     this.onImageLoad = this.onImageLoad.bind(this);
-
-    this.placeholder = null;
   }
 
-  onImageLoad() {
+  onImageLoad(): void {
```

Comment:
prefer the property [initializer syntax here](https://babeljs.io/docs/plugins/transform-class-properties/) (from [flow docs](https://flow.org/en/docs/react/events/)):
```
onImageLoad = (): mixed => {
```

Reply:
I agree on the return type, but why does it has to be a class property rather than a method?

Reply:
It means that you don;t have to bind `this` in ctor

Reply:
Also for consistency - we have started to use that everywhere else.

Reply:
This is wrong btw. It should be `() => void`